### PR TITLE
Add important note for setResult, vote, and finalize

### PR DESCRIPTION
**Pull request by dwalintukan**
image

image

image

### CODE REVIEW

**Comment by dwalintukan** [comment]
#247

**Comment by liviavinci** [comment]
This looks awesome!

File: src/languageProvider/locales/zh-Hans.json [hunk 1]
```diff
@@ -157,10 +152,11 @@
   "walletUnlockDialog.unlockWallet": "解锁钱包",
   "walletUnlockDialog.walletPassphrase": "钱包密码",
   "walletUnlockDialog.walletPassphraseRequired": "这个操作需要解锁您的钱包，请输入您的密码.",
+  "withdrawDetail.alreadyWithdrawn": "You have already withdrawn with this address.",
   "withdrawDetail.returnRate": "回报率：",
-  "withdrawDetail.reward": "回报",
-  "withdrawDetail.winningOutcome": "胜出选项",
-  "withdrawDetail.withdrawTo": "提现到",
+  "withdrawDetail.reward": "REWARD",
```
Comment (liviavinci) [inline src/languageProvider/locales/zh-Hans.json]: ???

File: src/languageProvider/locales/zh-Hans.json [hunk 2]
```diff
@@ -121,8 +115,8 @@
   "str.m": "分",
   "str.minutes": "分钟",
   "str.name": "名称",
-  "str.notEnoughBot": "您没有足够的BOT",
-  "str.notEnoughQtum": "您没有足够的QTUM",
+  "str.notEnoughBot": "Not enough BOT",
```
Comment (liviavinci) [inline src/languageProvider/locales/zh-Hans.json]: ???

**Comment by dwalintukan** [comment]
Sweet! 🎉

File: src/languageProvider/locales/en_US.json [hunk 3]
```diff
@@ -120,9 +113,8 @@
   "str.m": "m",
   "str.minutes": "Minutes",
   "str.name": "Name",
-  "str.notEnoughBot": "You don't have enough BOT",
-  "str.notEnoughQtum": "You don't have enough QTUM",
-  "str.minutes": "Minutes",
+  "str.notEnoughBot": "Not enough BOT",
```
Comment (liviavinci) [inline src/languageProvider/locales/en_US.json]: I feel like Not enough BOT sounds a bit cold

File: src/languageProvider/locales/zh-Hans.json (hunk 1 above)
Comment (dwalintukan) [reply src/languageProvider/locales/zh-Hans.json]: Hm i didnt change these.. I just ran the script.

File: src/languageProvider/locales/zh-Hans.json (hunk 2 above)
Comment (dwalintukan) [reply src/languageProvider/locales/zh-Hans.json]: Didn't change this either.

File: src/languageProvider/locales/en_US.json (hunk 3 above)
Comment (dwalintukan) [reply src/languageProvider/locales/en_US.json]: Didn't change this either. I think something is up with the script?

**Comment by dwalintukan** [comment]
I just rolled back one commit and didnt run the script. Looks like its changing some strings that are unintentional.

**Pull request merged**